src/components/app/ProfileSharing.js

```diff
@@ -58,11 +62,15 @@ const UploadingStatus = ({ progress }) =>
   </div>;
 
 UploadingStatus.propTypes = {
-  progress: PropTypes.number.isRequired,
+  progress: Number,
 };
```

Comment:
Here you can completely delete this propTypes definition, and instead type the argument of this functional component.

You can see https://github.com/DanielNill/perf.html/blob/4a7590769f01a50d175d3441f32743cf5fc9f178/src/components/app/Home.js#L22-L49 for a good example :)